controllers/databasecluster_controller.go

```diff
@@ -1043,6 +1049,17 @@ func (r *DatabaseClusterReconciler) reconcilePXC(ctx context.Context, req ctrl.R
 	if database.Spec.Proxy.Type == "" {
 		database.Spec.Proxy.Type = everestv1alpha1.ProxyTypeHAProxy
 	}
+	if database.Spec.Engine.Config == "" {
+		if database.Spec.Engine.Resources.Memory.Cmp(memorySmallSize) == 0 {
+			database.Spec.Engine.Config = pxcConfigSizeSmall
+		}
+		if database.Spec.Engine.Resources.Memory.Cmp(memoryMediumSize) == 0 {
+			database.Spec.Engine.Config = pxcConfigSizeMedium
+		}
+		if database.Spec.Engine.Resources.Memory.Cmp(memoryLargeSize) == 0 {
+			database.Spec.Engine.Config = pxcConfigSizeLarge
+		}
```

Comment:
Didn't get it. The `Cmp` returns 0 if the quantity is equal to the value. What if the memory is something in between? There were that intervals before, why it's not the case now?

Reply:
Frontend supports only 3 dimensions. The configuration might not work if memory has changed.